draft-dejong-remotestorage-06.txt

```diff
@@ -261,12 +262,13 @@ Internet-Draft              remoteStorage                  November 2015
     Note that the use of strong ETags prohibits changing the response
     body based on request headers; in particular, the server will not be
     able to serve the same document uncompressed to some clients and
-    gzipped when requested by the client, since the two bodies would not
-    be identical byte-for-byte.
+    compressed to other clients when requested, since the two bodies 
+    would not be identical byte-for-byte.
 
     Servers MAY support Content-Range headers [RANGE] on GET requests,
-    but whether or not they do SHOULD be announced through the <ranges>
-    variable mentioned below in section 10.
+    but whether or not they do SHOULD be announced both through the 
+    <ranges> variable mentioned below in section 10 and the HTTP 
+    'Accept-Ranges' response header.
```

Comment:
I think we're about to duplicate the HTTP spec again.

Also I don't see anything in [RFC 7233](https://tools.ietf.org/html/rfc7233) to support this claim:

>Yup. But the HTTP header is actually needed otherwise range request don't work.

Accept-ranges can be used by the client, but the client could also just attempt to do a range request, as non-supporting servers just ignore the header.